<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastrar_atendente</class>
 <widget class="QDialog" name="cadastrar_atendente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonCadastrar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>107</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:700;&quot;&gt;CADASTRAR ATENDENTE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>701</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>CPF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCPF"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditEmail"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Telefone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTelefone"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Whatsapp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditWpp"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSenha"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="checkBoxSGeral">
          <property name="text">
           <string>Secretaria Geral</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Medicos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMedicos"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>cadastrar_atendente</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
